patterns/2-structured/trusted-committer.md

```diff
@@ -127,6 +127,10 @@ This has been tried and proven successful at:
 
 - Nike
 - PayPal
+- Mercado Libre - We assign a section in the CONTRIBUTING file to inform who the TCs are.
+
+![image](https://user-images.githubusercontent.com/58035419/205659124-9fc7796d-3ee5-49b9-8c8c-1c7d24fa0488.png)
+
```

Comment:
```suggestion
- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.

![image](https://user-images.githubusercontent.com/58035419/205659124-9fc7796d-3ee5-49b9-8c8c-1c7d24fa0488.png)
```

We try to stay away from using "we" in the Known Instances, as each of them is referring to a different company, which makes the use of "we" potentially confusing.

Also did some other rewording, please commit if you are happy with this.